nextflow_schema.json

```diff
@@ -151,18 +151,26 @@
             "properties": {
                 "mle_design_matrix": {
                     "type": "string",
+                    "format": "file-path",
+                    "exists": true,
                     "description": "Design matrix used for MAGeCK MLE to call essential genes under multiple conditions while considering sgRNA knockout efficiency"
                 },
                 "rra_contrasts": {
                     "type": "string",
+                    "format": "file-path",
+                    "exists": true,
                     "description": "Comma-separated file with the conditions to be compared. The first one will be the reference (control)"
                 },
                 "count_table": {
                     "type": "string",
+                    "format": "file-path",
+                    "exists": true,
                     "description": "Please provide your count table if the mageck test should be skipped."
                 },
                 "library": {
                     "type": "string",
+                    "format": "file-path",
+                    "exists": true,
                     "fa_icon": "far fa-address-book",
                     "description": "sgRNA sequences and targeting genes"
                 },
```

Comment:
@LaurenceKuhl I have added the validation of these parameters, they must be files and they must exist. Could you check if it's correct? And if they have a pattern, like ending with `.csv` for example, we can also add it.